.github/workflows/go-setup/action.yml

```diff
@@ -4,10 +4,13 @@ description: "Sets up go environment and installs deps for go testing of armada"
 runs:
   using: "composite"
   steps:
+    # By setting cache:true, it enables caching of go modules, as long as the
```

Comment:
I think all the changes to this file are just ... leftover from how old this branch is. They can be reverted.